cmd/crio/main.go

```diff
@@ -243,8 +243,8 @@ func main() {
 
 		if config.CleanShutdownFile != "" {
 			// clear out the shutdown file
-			if err := os.Remove(config.CleanShutdownFile); err != nil {
-				// not a fatal error, as it could have been cleaned up
+			if err := os.Remove(config.CleanShutdownFile); err != nil && !os.IsNotExist(err) {
+				// continue anyways
```

Comment:
apparently the linter thinks this is a commented out `continue` :upside_down_face:

Reply:
Hah.  I removed it so we can continue